dispatcher/dispatcher.py

```diff
@@ -41,7 +43,12 @@ def main():
 
     sleep_period = int(config['PollFrequency'])
 
-    logger.info('Dispatcher starting up')
+    try:
+        commit = subprocess.run('git log -n 1 --pretty=oneline'.split(), capture_output=True).stdout.decode().split(' ')[0]
+    except Exception as e:
+        logger.debug(f'Couldn\'t get commit hash: {type(e)}, {e}')
+        commit = 'unknown'
```

Comment:
Could use https://github.com/XENONnT/daqnt/pull/79/files as well (also for related packages like daqnt)

Reply:
We could, but now that the biggest part of daqnt is here, the only things from daqnt that we still use are the logger, the bot, and the Mongo Client. I think it's much more important that we have a strict and accurate version for the dispatcher and hypervisor because these things affect uptime, while the bot and logger don't. Also the dispatcher isn't a module, and I don't see much reason to change this.